2021/OffSeason2021/src/main/cpp/commands/AutoDriveShoot.cpp

```diff
@@ -30,12 +30,14 @@ AutoDriveShoot::AutoDriveShoot(
     // Add your commands here, e.g.
     // AddCommands(FooCommand(), BarCommand());
     path = "driveForward";
+    double waitTime;
 
     AddCommands(
         IntakeDeploy(true),
+        frc2::WaitCommand(frc::SmartDashboard::GetNumber("AUTO_WaitTime", waitTime) * 1_s),
```

Comment:
Look at the API for frc::SmartDashboard::GetNumber.  The second parameter is used only if it can't find the tag "AUTO_WaitTime". We would just want that to be the value 0.0.